roles/common/python2/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# python-minimal can't be installed with apt_module as it requires python2 to run
+- name: Install python 2
+  raw: test -e /usr/bin/python || (apt -y update && apt install -y python-minimal)
+
+- name: Install pip
```

Comment:
The `install_dependencies` role should be usable here.

Reply:
done